postgres-connector/build.gradle

```diff
@@ -25,7 +34,18 @@ java {
 }
 
 jar {
-  from {
-    configurations.runtimeClasspath.collect { it.isDirectory() ? it : zipTree(it) }
-  }
-}
\ No newline at end of file
+
+// manifest {
+//        attributes(
+//                'Class-Path': configurations.compile.files.collect { "libs/$it.name" }.join(' ')
+//        )
+//    }
+
+    task copyJarToBin {
+        copy {
+            from 'build/libs/postgres-connector-2.5-SNAPSHOT.jar'
```

Comment:
be good to make this more general in future -- but for now I think the first step is updating the version. May be able to use wildcard